crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/match.py

```diff
@@ -263,3 +263,30 @@ def foo():
         # comment
     ):
         y = 1
+
+
+match pattern_match_class:
+    case Foo(
+            # own line
+            ):
+        ...
+
+    case Point2D(0, 0):
+        ...
+
+    case Point3D(x=0, y=0, z=000000000000000000000000000000000000000000000000000000000000000000000000000000000):
+        ...
+
+    case Bar(0, a=None, b="hello"):
+        ...
+
+    case FooBar(# leading
+# leading
+    # leading
+        # leading
+            0 # trailing
+# trailing
+    # trailing
+        # trailing
+            ):
+        ...
```

Comment:
Could you add something like
```python
match a:
    case A(
        b # b
        = # c
        2 # d
        # e
    ):
        pass
```
with the left hand side and the right hand side split of the assignment split by comment? I wanted to add an additional comment before the `b` but that's blocked by https://github.com/astral-sh/ruff/issues/6866